diff --git a/src/target.c b/src/target.c
index 2f9b5fa..d123736 100644
--- a/src/target.c
+++ b/src/target.c
@@ -564,6 +564,8 @@ int execute_cgi(t_session *session) {

	if ((wrap_cgi == false) && (session->cgi_type != fastcgi)) {
		check_file_exists = true;
+	} else if ((session->cgi_type == fastcgi) && (session->fcgi_server != NULL)) {
+		check_file_exists = false;
	} else if ((session->cgi_type == fastcgi) && session->fcgi_server->localhost
 #if defined(ENABLE_XSLT) || defined(ENABLE_MONITOR)
		&& session->host->show_index
